src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -175,6 +175,8 @@ def load_arguments(self, _):
     with self.argument_context('group deployment create') as c:
         c.argument('deployment_name', options_list=['--name', '-n'], required=False,
                    help='The deployment name. Default to template file base name')
+        c.argument('send_unmodified', action='store_true', is_preview=True,
+                   help='Support to send unmodified template content in deployment')
```

Comment:
Sounds good! I prefer adding `--handle-extended-json-format` to validate command at the same time.

Reply:
agreed

Reply:
> agreed

Given `validate` command is in another custom function, I think we can leave it in next sprint. Because it may be better if we can investigate the behavior in `create` command after this release and if no problem, I can add it in `validate` command and I also have enough time to do that.